apps/smart-contracts/token/utils/index.ts

```diff
@@ -153,3 +155,61 @@ export function generateDomainSeparator(
     SignTypedDataVersion.V4
   ).toString('hex')}`
 }
+
+// from UniswapV3 Periphery's test suite
+export async function getPermitSignature(
+  signer: SignerWithAddress,
+  token: ERC20PermitUpgradeable,
+  spender: string,
```

Comment:
```suggestion
  token: ERC20PermitUpgradeable,
  signer: SignerWithAddress,
  spender: string,
```
Confusing to have the token in between signer and spender